library/Rules/Email.php

```diff
@@ -1,10 +1,32 @@
 <?php
 namespace Respect\Validation\Rules;
 
+use Egulias\EmailValidator\EmailValidator;
```

Comment:
What will happen if you install `Respect/Validation` with `--no-dev` ? `Egulias\EmailValidator\EmailValidator` will not exists and it will break.